### DISCUSSION TITLE
when i use next, swr,prisma why response debug is 304

### DISCUSSION THREAD

**Discussion by itorz7** [prisma/prisma#14084] [Q&A]
<img width="566" alt="image" src="https://user-images.githubusercontent.com/30155968/176750088-76604f61-c05f-45ee-a337-d2857856190f.png">

but in postman 200 OK
<img width="349" alt="image" src="https://user-images.githubusercontent.com/30155968/176750131-bb9a5e5d-b74e-4686-bfee-e20eae696c95.png">

**Comment by austincrim** [comment]
Hey @itorz7 👋,

Are you using one of our templates?

If not, could you provide a minimal reproduction repository?

**Comment by itorz7** [comment]
@austincrim this here https://github.com/itorz7/next-swc-prisma

**Comment by itorz7** [comment]
i have one question how can i fix it ?, and Is there an interface for typescript ??
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/30155968/177052621-5e4b3c2c-e57c-490e-929c-856b1538d6d2.png">

**Reply by austincrim** [reply]
Do you need to define and import your own `PackageList` type? There's a mismatch between that defined type and the Prisma-generated `pacakge` type.

You can always import Prisma-generated types like so:
```ts
import type { package } from '@prisma/client
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
